source/docs/articles/sites/using-tmp-files.md

```diff
@@ -0,0 +1,39 @@
+---
+title: Using Temporary Files
+description: Instructions for using temporary files in distributed environments.
+category:
+- getting-started
+keywords: tmp, temp files, tmp files, temporary files, mulitiple application containers, distributed environments
+---
+
+##Temporary File Management Across Multiple Application Containers
+
+If you’re using multiple servers to process requests, you’ll need to set up a temporary directory that is shared between all environments.
+
+To share the temporary directory between all environments, use: `sites/default/files/tmp`
+
+##DROPs  
```

Comment:
I am not sure if we are still calling these DROPs. Not sure of the new standard convention though